modules/compute/compute.nix

```
@@ -2,6 +2,7 @@
 { ... }:
 {
   imports = [
+    ../generic/controllerHostEntry.nix
```

inconsistent style with other files
```suggestion
    ../generic/controller-host-entry.nix
```